ci-operator/step-registry/openshift/e2e/cert-rotation/test/openshift-e2e-cert-rotation-test-commands.sh

```diff
@@ -22,531 +22,7 @@ echo "### Copying test binaries"
-"[sig-storage] Secrets should be able to mount in a volume regardless of a different secret existing with same name in different namespace [NodeConformance] [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]"
-"[sig-storage] Secrets should be consumable from pods in volume [NodeConformance] [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]"
-"[sig-storage] Secrets should be consumable from pods in volume as non-root with defaultMode and fsGroup set [LinuxOnly] [NodeConformance] [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]"
-"[sig-storage] Secrets should be consumable from pods in volume with defaultMode set [LinuxOnly] [NodeConformance] [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]"
-"[sig-storage] Secrets should be consumable from pods in volume with mappings [NodeConformance] [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]"
-"[sig-storage] Secrets should be consumable from pods in volume with mappings and Item Mode set [LinuxOnly] [NodeConformance] [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]"
-"[sig-storage] Secrets should be consumable in multiple volumes in a pod [NodeConformance] [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]"
-"[sig-storage] Secrets should be immutable if `immutable` field is set [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]"
-"[sig-storage] Subpath Atomic writer volumes should support subpaths with configmap pod [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]"
-"[sig-storage] Subpath Atomic writer volumes should support subpaths with configmap pod with mountPath of existing file [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]"
-"[sig-storage] Subpath Atomic writer volumes should support subpaths with downward pod [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]"
-"[sig-storage] Subpath Atomic writer volumes should support subpaths with projected pod [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]"
-"[sig-storage] Subpath Atomic writer volumes should support subpaths with secret pod [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]"
-"[sig-storage] Subpath Container restart should verify that container can restart successfully after configmaps modified [Suite:openshift/conformance/parallel] [Suite:k8s]"
-"[sig-storage] Volumes ConfigMap should be mountable [Suite:openshift/conformance/parallel] [Suite:k8s]"
-"[sig-network] Proxy version v1 should proxy logs on node with explicit kubelet port using proxy subresource"
-"[sig-network] Proxy version v1 should proxy logs on node using proxy subresource"
+"[sig-cli] Kubectl logs logs should be able to retrieve and filter logs [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]"
```

Comment:
```suggestion
"[sig-cli] Kubectl logs should be able to retrieve and filter logs [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]"
```

Reply:
From the `e2e-cert-rotation-test` time and [content](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/49238/rehearse-49238-pull-ci-openshift-cluster-kube-apiserver-operator-master-e2e-metal-ovn-ha-cert-rotation-shutdown-90d/1762861827504476160), it seems that test is not enough.
```
INFO[2024-02-28T17:31:53Z] Running step e2e-metal-ovn-ha-cert-rotation-shutdown-90d-openshift-e2e-cert-rotation-test. 
INFO[2024-02-28T17:32:37Z] Step e2e-metal-ovn-ha-cert-rotation-shutdown-90d-openshift-e2e-cert-rotation-test succeeded after 44s. 
```

Reply:
I see there is a set of tests that cover a lot of `Conformance` tests, we can refer to 
https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift/microshift/e2e/openshift-conformance-reduced-aws/openshift-microshift-e2e-openshift-conformance-reduced-aws-commands.sh

Reply:
No, it worked correctly:
```
started: 0/1/1 "[sig-cli] Kubectl logs logs should be able to retrieve and filter logs [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]"

  I0528 17:32:26.604754  104455 reflector.go:351] Caches populated for *v1.Node from k8s.io/client-go@v0.29.0/tools/cache/reflector.go:229
  I0528 17:32:26.749294  104455 reflector.go:351] Caches populated for *v1.Pod from k8s.io/client-go@v0.29.0/tools/cache/reflector.go:229
passed: (9.1s) 2024-05-28T17:32:35 "[sig-cli] Kubectl logs logs should be able to retrieve and filter logs [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]"
```
[from test log](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift_release/49238/rehearse-49238-pull-ci-openshift-cluster-kube-apiserver-operator-master-e2e-metal-ovn-ha-cert-rotation-shutdown-90d/1762861827504476160/artifacts/e2e-metal-ovn-ha-cert-rotation-shutdown-90d/openshift-e2e-cert-rotation-test/build-log.txt)